.github/workflows/publish.yaml

```diff
@@ -0,0 +1,14 @@
+# A CI configuration to auto-publish pub packages.
+
+name: Publish
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    tags: [ 'v[0-9]+.[0-9]+.[0-9]+*' ]
+
+jobs:
+  publish:
+    if: ${{ github.repository_owner == 'dart-lang' }}
+    uses: devoncarew/firehose/.github/workflows/publish.yaml@main
```

Comment:
Hold on. Is this line running potentially-unreviewed code from your personal github repository using the credentials of this trusted github repository? This setup seems like a supply chain security violation. Can you move this repository to be owned by our organization so we can enforce mandatory review?

Reply:
Hmm, I prefer to think of it as a supply chain security opportunity. But yes, you're right to point this out.

My plan here was to set up a few repos with publishing automation (via that tool), iterate on the workflow until we knew we were happy with it, and then move the tool (w/ review and more tests) into a repo like dart-lang/ecosystem.

I'm happy to hold off on this PR until after the tool is moved, or come back and update this repo w/ the tool's new location, which ever you'd prefer. The time frame for iterating on and moving the tool is probably on the order of a few weeks.

Reply:
ptal, this is now updated to reference the dart-lang/ecosystem publishing workflow.